packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/dto/KeyBinding.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.diagrams.dto;
+
+import java.util.Objects;
+
+/**
+ * A key binding.
+ *
+ * @author gdaniel
+ */
+public record KeyBinding(boolean isCtrl, boolean isMeta, boolean isAlt, String key) {
```

Comment:
TODO @gdaniel move this in `sirius-components-collaborative` DTO package (this needs to be reusable for both trees and diagrams)